src/main.rs

```diff
@@ -15,15 +15,20 @@
 // Written by: Bobbbay Bobbayan
 
 use std::{thread, time, fs, env};
-
+use std::time::{SystemTime, UNIX_EPOCH};
 fn main() {
     let args: Vec<String> = env::args().collect();
 
+    let start = SystemTime::now();
+    let since_the_epoch = start
+        .duration_since(UNIX_EPOCH)
+        .expect("Time went backwards");
+    
     let mut time = 10; // Set time in ms
     
     // If args are larger than 1 then parse them 
     if(args.len() > 1) {
-      if(args[1] == "-t") {
+      if(args[1] == "-e") {
         time = args[2].parse::<u64>().unwrap();
       }
     }
```

Comment:
I would really use a switch case here.

Reply:
cba to learn rust i hate the language googled it for like 5 minutes dont like the look of switches in rust

Reply:
Haha, rust switches are better than you think!

```rust
match number {
  // Match a single value
  1 => println!("One!"),
  2 | 3 | 5 | 7 | 11 => println!("This is a prime"),
  _ => {
    println!("Default");
    println!("Second command");
  }
}
```

Or something along those lines.